parts/kubernetesmastervars.t

```diff
@@ -179,7 +178,7 @@
     "agentWindowsSku": "2016-Datacenter-with-Containers",
     "agentWindowsVersion": "2016.0.20170127",
     "singleQuote": "'",
-    "windowsCustomScriptSuffix": " $inputFile = '%SYSTEMDRIVE%\\AzureData\\CustomData.bin' ; $outputFile = '%SYSTEMDRIVE%\\AzureData\\CustomDataSetupScript.ps1' ; Copy-Item $inputFile $outputFile ; Start-Job -command { Invoke-Expression('{0} {1}' -f $outputFile, $arguments) } ; "
```

Comment:
501 is removing the Start-Job: https://github.com/Azure/acs-engine/pull/501/files#diff-f18f17bd6f84d54a111be39c8fe59d54R182

I guess I don't even understand why this diff is showing up here.